core/src/server.rs

```diff
@@ -60,5 +61,14 @@ async fn show_config(Extension(config): Extension<Config>) -> impl IntoResponse
 }
 
 async fn show_version() -> impl IntoResponse {
-    return format!("Serial Keel v{}\n", env!("CARGO_PKG_VERSION"));
+    let mut version = Version::new(0, 0, 0);
+    version = Version::parse(env!("CARGO_PKG_VERSION")).unwrap_or(version);
```

Comment:
I suggest we do `let version = Version::parse(...).expect("The server version should always be semver parsable")`.
If this is _not_ the case, then we have a bug, and then we fix the bug until it works as expected.
(But in this case it will probably never fail).

Reply:
done :+1: